src/GDGUkraine/controller.py

```diff
@@ -42,6 +42,7 @@ def index(self, **kwargs):
                                           filtered=True))
 
     @cherrypy.expose
+    @cherrypy.tools.authorize()
```

Comment:
No requests reach this address, since main admin template is served as static and is then used by AngularJS